support-frontend/assets/pages/contributions-landing/components/ContributionAmount.jsx

```diff
@@ -167,13 +140,14 @@ function withProps(props: PropTypes) {
     formatAmount(currencies[props.currency], spokenCurrencies[props.currency], { value: max.toString() }, false);
   const otherAmount = props.otherAmounts[props.contributionType].amount;
 
-  const renderContribAmountChoiceCards = () => (
+  const renderYellowChoiceCards = () => (
     <>
       <ChoiceCardGroup
         name="amounts"
       >
         {validAmounts.map((amount: Amount) => (
           <ChoiceCard
+            cssOverrides={yellowChoiceCard}
```

Comment:
It looks like the only difference between `renderYellowChoiceCards` and `renderControl` is the `cssOverrides` prop.
If so we can have a single function, e.g.
`const renderChoiceCards = (cssOverrides: SerializedStyles | null) => (...)`
and
`{props.choiceCardsVariant === 'yellow' ? renderChoiceCards(yellowChoiceCard) : renderChoiceCards()}`